lib/chef/provider/dsc_resource.rb

```diff
@@ -92,6 +101,10 @@ def supports_refresh_mode_enabled?
         Chef::Platform.supports_refresh_mode_enabled?(node)
       end
 
+      def module_usage_valid?
+        !(!@module_name && @module_version) ? true : false
```

Comment:
Since the left hand side is already going to eval to `TrueClass` or `FalseClass`, I don't think we need the ternary expression at the end.